content/docs/2.13/scalers/memory.md

```diff
@@ -13,7 +13,7 @@ go_file = "cpu_memory_scaler"
 
 ### Prerequisites
 
-KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include `requests` (at a minimum).
+KEDA uses standard `cpu` and `memory` metrics from the Kubernetes Metrics Server, which is not installed by default on certain Kubernetes deployments such as EKS on AWS. Additionally, the `resources` section of the relevant Kubernetes Pods must include at least one of `requests` or `limits`.
```

Comment:
this was added to CPU scaler but we missed it for the memory scaler docs, this just brings the two pages to parity matching the implementation

https://github.com/kedacore/keda-docs/pull/1296#discussion_r1455677499